src/handlebar-helpers.js

```diff
@@ -65,6 +65,12 @@ const helpers = {
     }
     const hash = crypto.createHash('sha256').update(data).digest('hex');
     return hash;
+  },
+  concat: function(arr){
```

Comment:
is there a reason you want to pass in an array as opposed to accepting an arbitrary number of parameters?
ie.
```
concat: function(...arr){
  return ''.concat(...arr)
}

HandlebarHelper.concat("Hello ", "World")
```

Reply:
or better yet, accept both
```
concat: function(...arr){
  let flat = arr.flat();
  return ''.concat(...flat); 
}
```

Reply:
> is there a reason you want to pass in an array as opposed to accepting an arbitrary number of parameters?
> ie.
> 
> ```
> concat: function(...arr){
>   return ''.concat(...arr)
> }
> 
> HandlebarHelper.concat("Hello ", "World")
> ```

@alewitt2 considering case where more than 2 strings needs to join .... feel free to change